src/test/regress/sql/pg15.sql

```diff
@@ -337,8 +337,55 @@ DELETE FROM PKTABLE WHERE id = 1 OR id = 2;
 SELECT * FROM FKTABLE ORDER BY id;
 
 
--- Clean up
 \c - - - :master_port
 
+create table reservations ( room_id integer not null, booked_during daterange );
```

Comment:
It looks like we forget to set `search_path` after a `\c` command. This should be fine after a rebase as I already fixed that in #6306